docs/site/downloads/cldr-47.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: CLDR 47 Release Note
+date: 2025-03-13
+tag: release-47
+json: 47.0.0
 ---
 
-# CLDR 47 Release Note
-
 | No. |    Date    | Rel. Note |  Data  |  Charts  | Spec |   Delta  | GitHub Tag | Delta DTD | CLDR JSON |
 |:---:|:----------:|:---------:|:------:|:--------:|:------------:|:---:|:----------:|:---------:|:---------:|
-|  47 | 2025-04-~~XX~~ | [v47](/index/downloads/cldr-47) | ~~[CLDR47](https://unicode.org/Public/cldr/47/)~~ | [Charts47](https://unicode.org/cldr/charts/dev) | [LDML47](https://www.unicode.org/reports/tr35/proposed.html) | [Δ47](https://unicode-org.atlassian.net/issues/?jql=project+%3D+CLDR+AND+status+%3D+Done+AND+resolution+%3D+Fixed+AND+fixVersion+%3D+%2247%22+ORDER+BY+priority+DESC) | ~~release-47~~<br>[release-47-beta2](https://github.com/unicode-org/cldr/releases/tag/release-47-beta2) | [ΔDtd47](https://www.unicode.org/cldr/charts/dev/supplemental/dtd_deltas.html) | ~~47.0.0~~<br>[47.0.0-BETA2](https://github.com/unicode-org/cldr-json/releases/tag/47.0.0-BETA2) |
-
-<span style="color:red; font-weight: bold;">This is an beta version of CLDR v47.</span>
+|  47 | 2025-03-13 | [v47][] | [CLDR47][] | [Charts47][] | [LDML47][] | [Δ47][] | [release-47][] | [ΔDtd47] | [47.0.0][] |
```

Comment:
I'm wondering about the empty brackets "[]". I did see below "[v47]: /downloads/cldr-47" and so forth.

There is documentation here: https://github.com/adam-p/markdown-here/wiki/markdown-cheatsheet#links

I don't see a mention of empty brackets though. All these links have empty brackets except for [ΔDtd47]. Are they optional?

Reply:
https://www.markdownguide.org/basic-syntax/#reference-style-links

Reply:
> `Or leave it empty and use the [link text itself].`

we actually can remove the `[]` according to that.